src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/model/dto/Document.java

```diff
@@ -30,6 +31,12 @@ public Document(String id, String groupId, List<String> tokenizedTitle, List<Str
     this.originalScore = originalScore;
   }
 
+  /**
+   * No-args constructor used to deserialize.
+   */
+  public Document() {
+  }
+
```

Comment:
if no return type, should it be void?

Reply:
It's a constructor. 

It doesn't return anything so much as it initializes the object (in this case, it initializes all fields to `null`).

Reply:
I see, thanks!